### ISSUE TITLE
netatmo weather station  has stopped communicating

### ISSUE THREAD

**Issue by kitopopo** [domoticz/domoticz#4477]
Dear friends,

Throughout the afternoon today we have lost communication with the Netatmo weather station.
I have tried with domoticz previous beta versions and the problem persists. i can see in the domoticz main log

```
Error: Netatmo: Error connecting to Server...
Error: Netatmo: cloud data for module [Externo] no longer updated (module possibly disconnected)
Error: Netatmo: cloud data for module [habitacion matrimonio] no longer updated (module possibly disconnected)
Error: Netatmo: cloud data for module [pluviometro kike] no longer updated (module possibly disconnected)
Error: Netatmo: cloud data for module [Indoor] no longer updated (module possibly disconnected)
Error: Netatmo: cloud data for module [] no longer updated (module possibly disconnected)
Error: Netatmo: cloud data for module [] no longer updated (module possibly disconnected)
```

Best regards

**Comment by kiddigital** [comment]
@kitopopo , please use the forum for these questions.

**Comment by gizmocuz** [comment]
Maybe the module is not connected, or it got a problem with WiFi.
Indeed first create an issue on the Forum, if more people have this problem then please reopen this issue

**Comment by kitopopo** [comment]
> Tal vez el módulo no esté conectado o tenga un problema con WiFi.
> De hecho, primero cree un problema en el Foro, si más personas tienen este problema, vuelva a abrir este problema.

dear friend @gizmocuz ,

The weather station did not have any problem with the wifi , the netatmo android application it worked correctly 
through wifi and without it and as I wrote it was not a problem with the domoticz beta, surely more people have had this problem for several hours. As I have already mentioned, I have done several tests before opening the issue, and it was probably due to the netatmo server or netatmo API. 

(I thought that it could be an api problem since netatmo has now merged with several companies like btizino and legrand)

Now it works fine, it seems that netatmo has solved the problem, but I insist again, the problem was either Netatmo or domoticz. 
I apologize for not being able to find this out. When I open an issue is when people start commenting on the issue that they have the same problem, i don't usually leave issues open if things work again,  I am responsible for closing them. 

greetings and thanks  for your colaboration

**Comment by gizmocuz** [comment]
Thats why it is best to use the forum!

**Comment by kitopopo** [comment]
I will do it next time @gizmocuz ,

But there in the forum I posted a problem that the virtual switches did not work on the floorplan (push on button) when changing the default icon and adding another favorite icon and that person opened an issue on github about a problem that I had discovered, I do not know if you remember the problem. For practical purposes it is the same result and the problem 
not resolved yet.

https://www.domoticz.com/forum/viewtopic.php?f=6&t=34269&p=258912#p258912

https://github.com/domoticz/domoticz/issues/4413


But don't worry I'll use the forum first. I apologize again.

regards
